docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,67 @@
+.NET Core
+=========
+
+.NET Core is a general purpose app platform developed by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supports Windows, OS X and Linux and intends to support a wide variety of CPUs and OSes over time. It can be used in device, cloud and embedded/IoT scenarios. We expect it to be extended to support additional scenarios over time (e.g. WebAssembly), both by Microsoft and the community.
+
+.NET Core is composed of a few parts:
+
+- A [.NET runtime](https://github.com/dotnet/coreclr) that can be used for a wide variety of app types
+- A set of [framework libraries]((https://github.com/dotnet/corefx)) that run on CoreCLR and other runtimes. 
```

Comment:
Hm, confusing a bit. Maybe something like "A set of higher level libraries that run on .NET runtime and other .NET Core compatible runtimes" or similar?